Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,6 +37,40 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#define WINAPIV
```

Comment:
WINAPIV doesn't make sense in Linux. Please create OSAPI for both Windows and Linux